src/app/markers.ts

```diff
@@ -68,6 +80,8 @@ export function roundMarkerFromRoundFlower(
     isBlooming: flower.isBlooming,
     isNest: false,
     canVisit,
+    incompatibleFlower,
+    tip: lastVisitedIncompatible ? 'I don\'t like this flower. No Pollen collected' : null
```

Comment:
```suggestion
    tip: lastVisitedIncompatible ? `${bee.name}s can't collect pollen from this flower` : null
```

KK, Joe, and I discussed some of the phrasing stuff and concluded that we want to avoid the explicit first-person phrasing.